scripts/entrypoint.sh

```diff
@@ -18,3 +18,5 @@ python manage.py migrate
 echo 'Collecting Static Files...'
 python manage.py collectstatic --no-input
 
+echo 'Creating Groups...'
+python maange.py create_groups
```

Comment:
```suggestion
python manage.py create_groups
```

Reply:
that's what I get for doing shit on the train